fixtures/manufacturers.json

```diff
@@ -20,6 +20,9 @@
     "name": "AFX",
     "website": "https://www.afx-light.com"
   },
+  "algo-suena": {
+    "name": "Algo Suena"
```

Comment:
I see "AUVI LIGHTS" in the fixture name, and there is an [Auvi Lights](https://www.auviprod.com/auvi-lights/), but this fixture isn't listed there now, so maybe it's a discontinued model.

The only reference I can find is [this retailer](https://www.fmelectronic.net/product/led-wash-z19-zoom-moving-head-rgbw-auvi-ligths/).  There's no manual available online, and not even a brand name printed on the fixture.

I can't find where any of the names "Algo Suena", "ELCARMEN", or "TELCARMEN" come from.